src/test/java/remer/sudoku/SudokuTest.java

```diff
@@ -0,0 +1,44 @@
+package remer.sudoku;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class SudokuTest {
+    @Test
+    public void getErrors() {
+        int[][] exampleBoard =
+                {
+                        /* CORRECT BOARD
+                        {5, 3, 4, 6, 7, 8, 9, 1, 2},
+                        {6, 7, 2, 1, 9, 5, 3, 4, 8},
+                        {1, 9, 8, 3, 4, 2, 5, 6, 7},
+                        {8, 5, 9, 7, 6, 1, 4, 2, 3},
+                        {4, 2, 6, 8, 5, 3, 7, 9, 1},
+                        {7, 1, 3, 9, 2, 4, 8, 5, 6},
+                        {9, 6, 1, 5, 3, 7, 2, 8, 4},
+                        {2, 8, 7, 4, 1, 9, 6, 3, 5},
+                        {3, 4, 5, 2, 8, 6, 1, 7, 9}
+                         */
+
+                        // 6 ERRORS IN BOARD
+                        {1, 3, 4, 6, 7, 8, 9, 1, 2},
+                        {6, 7, 2, 1, 3, 5, 3, 4, 8},
+                        {1, 9, 8, 3, 4, 2, 5, 6, 7},
+                        {8, 5, 9, 7, 6, 1, 4, 2, 3},
+                        {2, 2, 6, 8, 5, 3, 7, 9, 1},
+                        {7, 9, 3, 9, 2, 4, 8, 5, 6},
+                        {9, 6, 1, 5, 3, 7, 6, 8, 4},
+                        {7, 8, 7, 4, 1, 9, 6, 3, 5},
+                        {3, 4, 5, 2, 8, 6, 1, 7, 9}
+                };
+
+        Sudoku sudoku = new Sudoku(exampleBoard);
+        List<String> errors = sudoku.getErrors();
+
+        // Print errors
+        for (String error : errors) {
+            System.out.println(error);
+        }
```

Comment:
This isn't the correct way to test for errors.

You must do an assert in your test or else it isn't a test